packages/beacon-node/src/chain/errors/lightClientError.ts

```diff
@@ -22,7 +22,7 @@ export class LightClientError extends GossipActionError<LightClientErrorType> {}
 // Errors for the light client server
 
 export enum LightClientServerErrorCode {
-  RESOURCE_UNAVAILABLE = "RESOURCE_UNAVALIABLE",
+  RESOURCE_UNAVAILABLE = "RESOURCE_UNAVAILABLE",
```

Comment:
are error codes parsed/checked somewhere? I did not find a reference to `RESOURCE_UNAVALIABLE` anywhere in the code but if this code is used somewhere else should probably keep the typeo.